app/controllers/katello/api/v2/content_export_incrementals_controller.rb

```
@@ -87,5 +115,12 @@ def find_exportable_organization
         throw_resource_not_found(name: 'organization', id: params[:organization_id])
       end
     end
+
+    def find_exportable_repository
+      @repository = Repository.find_by_id(params[:id])
+      if @repository.blank? || !@repository.organization.can_export_content?
```

May want to throw a separate error for can_export_content? evaluating to false